<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CColorEditWidget</class>
 <widget class="QWidget" name="CColorEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>204</width>
    <height>103</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="0" column="0">
    <widget class="QSlider" name="rHorizontalSlider">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="rSpinBox">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::UpDownArrows</enum>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="browsePushButton">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QSlider" name="gHorizontalSlider">
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="gSpinBox">
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2" rowspan="3">
    <widget class="QWidget" name="graphicsWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>75</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QSlider" name="bHorizontalSlider">
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="bSpinBox">
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QSlider" name="aHorizontalSlider">
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="aSpinBox">
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>255</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rHorizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>rSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gHorizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>gSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bHorizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>bSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aHorizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>aSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>rHorizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>gHorizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>bHorizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>aHorizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
